samples/wifi/shell/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+/* Wi-Fi needs more flash (1M) and RAM (512K) */
+&cpuapp_slot0_partition {
+	reg = <0xa6000 DT_SIZE_K(1024)>;
+};
+
+&cpuapp_data {
+	reg = <0x1000 DT_SIZE_K(512)>;
```

Comment:
doesn't this exceed `cpuapp_ram0x_region` region (260K)?

Reply:
cc: @57300

Reply:
Yes, I had the override for `cpuapp_ram0x_region` here but I guess it's missed in rebase, will fix, thanks.

Reply:
rejigged the memory layout to suit Wi-Fi.